Comp2022/src/main/include/commands/IntakingAction.h

```diff
@@ -17,6 +17,8 @@
 
 #include <frc2/command/CommandHelper.h>
 #include <frc2/command/ParallelCommandGroup.h>
+#include <frc2/command/SequentialCommandGroup.h>
+#include <frc2/command/WaitUntilCommand.h>

```

Comment:
@jmullins16 these include are only used in the cpp file. At one point in time, includes in an h file were comparatively very expensive because they would need to be re-parsed by every build that includes IntakingAction.h - do you have any idea whether this is still the case? This might be the kind of problem that clang has cached out of existence